README.md

```diff
@@ -519,6 +519,30 @@ func isError(error: Error?) -> Bool {
 }
 ```
 
+If there's a case where you want to enter a scope if AT LEAST ONE item is non-nil then use the `??` operator.
+
+```swift
+if let _ = item ?? item1 {
```

Comment:
I wanted a way to enter a scope if at least one object in a series of objects is not nil using `if let`. In that case should we support just checking directly for nil?